Mathlib/Combinatorics/SimpleGraph/Copy.lean

```diff
@@ -229,4 +279,51 @@ lemma free_bot (h : A ≠ ⊥) : A.Free (⊥ : SimpleGraph β) := by
 
 end Free
 
+/-!
+#### Induced containment
+
+A graph `H` *inducingly contains* a graph `G` if there is some graph embedding `G ↪ H`. This amounts
+to `H` having an induced subgraph isomorphic to `G`.
+
+We denote "`G` is contained in `H`" by `G ⊴ H` (`\triangle_left_eq`).
+-/
+
+/-- A simple graph `G` is contained in a simple graph `H` if there exists an induced subgraph of `H`
+isomorphic to `G`. This is denoted by `G ⊴ H`. -/
+def IsIndContained (G : SimpleGraph V) (H : SimpleGraph β) : Prop := Nonempty (G ↪g H)
+
+@[inherit_doc] scoped infixl:50 " ⊴ " => SimpleGraph.IsIndContained
+
+protected lemma IsIndContained.isContained : G₁ ⊴ G₂ → G₁ ⊑ G₂ := fun ⟨f⟩ ↦ ⟨f, f.injective⟩
+protected lemma Iso.isIndContained (e : G ≃g H) : G ⊴ H := ⟨e⟩
+protected lemma Iso.isIndContained' (e : G ≃g H) : H ⊴ G := e.symm.isIndContained
+
+protected lemma Subgraph.IsInduced.isIndContained {G' : G.Subgraph} (hG' : G'.IsInduced) :
+    G'.coe ⊴ G :=
+  ⟨{  toFun := (↑)
+      inj' := Subtype.coe_injective
+      map_rel_iff' := hG'.adj.symm }⟩
```

Comment:
```suggestion
  ⟨{ toFun := (↑)
     inj' := Subtype.coe_injective
     map_rel_iff' := hG'.adj.symm }⟩
```
We don't seem to use double spaces most of the time, although I see why you wanted to here

Reply:
Oh in fact that's a choice mathport wanted for me (and recently I have witnessed that that's what the blue lightbulb wants for you too)

Reply:
I'm not sure mathport's style choices are necessarily the ones we should follow in mathlib :)